airbyte-cdk/python/airbyte_cdk/sources/declarative/requesters/error_handlers/default_error_handler.py

```diff
@@ -19,6 +19,7 @@
 class DefaultErrorHandler(ErrorHandler):
     """
     Default error handler.
+
```

Comment:
[pep 257](https://peps.python.org/pep-0257/) says there should be an empty line between the summary and the details :shrug:

Reply:
https://www.youtube.com/watch?v=wf-BqAjZb8M